### DISCUSSION TITLE
Delete other buffers when opening a file from tree?

### DISCUSSION THREAD

**Discussion by volgar1x** [nvim-tree/nvim-tree.lua#1105] [Q&A] [answered]
Is it possible in the current state of the plugin to delete other buffers when opening a file from tree?

**Comment by volgar1x** [comment, marked as answer]
I wrongly thought that buffers were tab-isolated, which kind of defeat my original goal.

**Reply by kyazdani42** [reply]
Indeed, they are not :)
